presto-tests/src/test/java/com/facebook/presto/tests/tpch/TpchQueryRunner.java

```diff
@@ -45,32 +45,47 @@ public static DistributedQueryRunner createQueryRunner(Map<String, String> extra
         DistributedQueryRunner queryRunner = TpchQueryRunnerBuilder.builder()
                 .setExtraProperties(extraProperties)
                 .setResourceManagerEnabled(true)
+                .setCatalogServerEnabled(true)
```

Comment:
Let's leave this method as it is. Maybe create another method with a parameter `boolean catalogServerEnabled` to determine if it should be enabled or now.